modules/govuk_scripts/files/usr/local/bin/govuk_app_console

```diff
@@ -9,19 +9,19 @@
 
 set -eu
 
-if [ "$#" -ne "1" ] || [ "$1" = "-h" ]; then
-  echo "Usage: $(basename "$0") app_name" >&2
+if [ "$#" -gt 0 ] || [ "$1" = "-h" ]; then
```

Comment:
This says if the number of provided arguments is greater than 0, then show the help message right? If so, won't that happen always?

Reply:
Agreed.  It also won't work because the LHS is a string but the RHS is an integer.

After [some research](https://stackoverflow.com/questions/18568706/check-number-of-arguments-passed-to-a-bash-script) I'd recommend `[[ ($# -ne 1 || $# -ne 2) || "$1" == "-h" ]]`, bearing in mind I wrote the not-amazing original clause.